scripts/validate-commits.sh

```
@@ -0,0 +1,298 @@
+}
+
+# Format error information for invalid commits (returns formatted string)
+format_commit_error() {
+  local commit="$1"
+  local author_name="$2"
+  local author_email="$3"
+  local commit_msg="$4"
+  local error_type="$5"
+  local description="$6"
+  
+  local short_sha=$(echo "$commit" | cut -c1-8)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)